### DISCUSSION TITLE
Mimic Status

### DISCUSSION THREAD

**Discussion by Digisembler** [ISS-Mimic/Mimic#228] [General]
Status: 21-Jul-2021

@juggler2000 @GodspeedDiscvry @cactus314 @SpaceTowel @streadgold 

Transferring our weekly status here, so we can easily convert new items into issues.

Reminder that nothing sensitive should be posted here, but since we have non-employees in our email chain now nothing sensitive should be on the email either. 

Moving forward, I think our weekly Mimic meetings should begin with (or include) a review of the github issues page, to quickly status each issue (possibly excluding long term issues). We can document all our notes here in the discussions and easily transfer items to the issues page. (this is also accessible anytime without needing to check work email).

Over the weekend we transferred MIMIC ONE to Patrick's house for him to use as a reference for his build. In the setup we burned out another Metro on the Stbd side. This was documented in issue #222. Patrick has ordered the PCBs and we will see how they turn out. Will need to update the PCB order process with pictures send to JLCPCB so future users don't need to interact with their engineering team to place components. (make issue for this)

Today we went through the issue list and closed several old or duplicate or fixed issues. @SpaceTowel is working on updating the 3D printing table in the wiki which should be finished momentarily

And Craig (I mean me I'm totally Craig) printed a new IROSA test piece that looks like it will work great with our arrays

**Comment by SpaceTowel** [comment]
8/4/2021:  Tag up summary

1. [@cactus314  + @streadgold] Switching neopixle control board to metro mini
2. [@cactus314+ @juggler2000 + @streadgold]  Find a way to troubleshoot the burnout metro board. Patrick to send the burned out board via Wendy Airmail - ETA, tomorrow! [Bryan] Drop off at Sam's desk. Sam will start the troubleshoot
3. [@streadgold ] to update silk screen labels to adjust viewability after screw terminal is installed  (#241 + #242)
4. [@cactus314 ] to order new parts after the above is done by Sam ticket .  To keep mimic as updated as possible
5. [@Digisembler ] to send iROSA + MLM fusion model to Sam
6. [@SpaceTowel ] to add iROSA + MLM model to print table (wiki), after Sam has link ready
7. [@GodspeedDiscvry ] to design iROSA vinyl image, using double vinyl overlay, aka no paint.  
8. [@SpaceTowel ] to cut iROSA vinyl
9. [@streadgold  + @cactus314 ] to work NumPy intermittent issue troubleshooting.  [Patrick] to find out NumPy version number to start
10. [All] 8/19/21 (Thursday) : Guest presenter at re:3D monthly Meetup, via zoom, 20 min pitch
    - [ ] Short pre-recorded vieo clip/still
    - [ ] @cactus314  pointing at STEM Mimic live

**Comment by SpaceTowel** [comment]
8/11/2021: Tag up summary

1. Run through previous week summary. 
2. GitHub issue check
3. Wiki - Print table, remove the 2 'Posted' columns.   Rename '3MF' as 'Units' column.  @streadgold to add new issue.
4. Airlock print - great progress. 
5. In regards to ticket #254.  @GodspeedDiscvry to create views to show magnetic locations and polarity.    For Reference: https://historicspacecraft.com/ISS.html ;  https://historicspacecraft.com/Diagrams/S/ISS_Visiting_Vehicles_10Apr2016_RK2016.jpg
6. Re:3D virtual Open House presentation Aug 19, 6:30pm CT. via zoom. 20 min pitch.  more details »
     Join Zoom Meeting:   https://us06web.zoom.us/j/81394788701?pwd=S0wzY0Y4SFhnMWJPb2x2SXBNVldDZz09
     Meeting ID: 813 9478 8701
     Passcode: _KZSK91m_x
     One tap mobile
    +13462487799,,81394788701#,,,,*8735504351# US (Houston)
    +17207072699,,81394788701#,,,,*8735504351# US (Denver)

7. Opportunity to present Mimic to Bangladesh STEM Club - ~50 students 10-16 yrs old.  Choose a weekday morning as desired as that’s evening for them. Local organizer can come to BAB and help facilitate.  Seems like a 20-30 min pitch, then Q&A Contact was via Creatorspace
8. @juggler2000  to dissemble and measure the screw length (S3 + P3), in order to add Micro gear dc motor screws to BOM, #226 & #204.  (5mm)
9. Blender #125, still need components to split, look for not checked items
9. Closed 2 issues, yay! #202 & #171

**Comment by cactus314** [comment]
Personal progress update:
- With the exception of adding the shutters via hinges to the cupola and waiting on the STL for the MLM airlock (Issue #253), the 3D-printed USOS and ROS segments of the Mimic model I am building are complete (I think).  I'm looking forward to getting the updated Z1 model from @streadgold when he's ready to release it, along with the Dragon model.
![image](https://user-images.githubusercontent.com/10385574/130794197-e29715dc-7cc5-4695-b766-c272c8c8b744.png)
- HDMI breakout and IEA boards with reworked silkscreens have arrived from JCLPCB and look okay.  I should be able to start wiring now, but I'm still trying to work out the proper workflow/order of operations in my head.  I may want to wait until I have all the truss pieces printed, which potentially means waiting for a new S0 design (see Issue #255).
- @Digisembler has sent me the STLs for the iROSA model, but I haven't had a chance to try printing them yet.  There are 3 pieces, but I haven't really looked at them to figure out how they glue together for the final assembly.  I hope to print, assemble and install them on the STEM model this weekend (though without the vinyl overlays).
- Uploaded new frame mount for the HDMI breakout board and closed Issue #243.
- Issue #187: Have looked at different options for going to a smaller board for controlling the Neopixels,  I think that sticking with a Metro 0 Express (same as used for motor control), while overkill, is the most reliable, simplifies the BOM, reduces soldering and doesn't add a huge cost differential.  If there is agreement to keeping the Metro 0 for the Neopixels, Issue #187 can be closed, and I'll update the BOM as part of Issue #265.
- Issue #244: Have a .img.gz file tested and ready to be uploaded to SourceForge.  I'll get it to @streadgold via sneakernet and then this issue can be closed.
- Issue #254: With help from @Digisembler, an [updated spreadsheet](https://github.com/ISS-Mimic/Mimic/blob/master/docs/ISS%20Mimic%20Magnet%20Mapping_082121.xlsx) was created to list the magnet orientations for each module in the USOS and ROS.  It has been uploaded to the documents directory and will be used to generate a table in the build guide.  It can also be used as a reference for creating whatever graphics are desired for supporting documentation.
- Issue #256: I've put in most all of the acronyms I've run across in the physical build and in the Mimic GUI, along with links to Wikipedia where appropriate.  There are still a couple of them that I'm not 100% sure on (though I could guess), so I'll let someone else fill those in, and then this issue can be closed.
- Issue #265: After I update the Neopixel controller board, I'll simply be waiting on the marked magnet from K&J to make sure it's not too strong for use for installing the magnets in the modules as I envision.  Then I'll close out that issue.

**Reply by ISS-Mimic** [reply]
I believe the Z1 design is finished btw, just need to test print a few more times

**Comment by SpaceTowel** [comment]
8/25/2021: Tag up summary

1. Gone through @cactus314  working status
2. Closed #263 + #253 by @Digisembler 
3. Freeman Library build community-built Mimic deferred to Spring 2022, due to COVID
4. @Digisembler emailed views for magnetic map to @cactus314

**Comment by SpaceTowel** [comment]
9/1/2021: Tag up summary

1. Considered Galveston Children's Museum event - Team Mimic decided to not participate. 
2. Moving Cricut maker to CreatorSpace in the next few weeks.  Can iROSA image ready in the next couple of weeks?
3. Convert Mimic lite from laser-cut wood to 2D puzzle (printed with 3D printer).  
4. Updated images for magnetic mapping #254
5. Updated acronym list #256

**Comment by SpaceTowel** [comment]
9/8/2021: Tag up summary

1. Talked #269  - @streadgold will send model of PCB assembly to @Digisembler 
2. Created iROSA vinyl using Inkscape!!!!   Will see if it will cut properly next. 
3. ISS mimic model can be opened and viewed via Fusion 360 by everyone attached to the project.

**Reply by ISS-Mimic** [reply]
Reminder for everyone - https://myhub.autodesk360.com this is the website from which anyone added to the Mimic fusion project should be able to view and comment on the models (might also be able to download/open in desktop through there?)

**Comment by streadgold** [comment]
9/29/2021: Tag up summary

Talked about #269  some more, closed several issues #270 #271 #267

**Comment by streadgold** [comment]
3/9/2022

Talked upcoming events on 3/23 and 3/28

Need to have SAWs finished by then

Also need to replace the outboard truss and fix SASA by then

And we got our RNASA recognition award thing!

**Comment by wendi881** [comment]
I am very interested in what you are doing building the ISS Mimic. I would really like to try to do something like that for my classroom. I don't understand most of the files here. Is there an easy way to figure out all of the parts I would need to get or print? How much does one need to know to build the ISS Mimic? Thanks for any help you can give.

**Reply by Digisembler** [reply]
Hi wendi881!  Thanks for noticing us and being interested in our project, and thrilled that you would like to try this for your classroom.  Admittedly, github can be a bit scary at first, hang in there!  At the top of the page (this one even), if you were to click on the Wiki text, it will take you over to the pages to read up more and how to get started (the menu on the right of that page has other pages to look at).  Mostly there is downloading the stereolithography files (stl or 3d printer files), the bill of materials (BOM) to order parts for non-printed pieces, then printing and assembling (I'm simplifying it a bit... but it's doable!).  Take a look.  If this seems daunting still, we have a smaller version that's in work.  We do have an educator helping us to bridge the gaps between us and classrooms, keep the questions coming!

**Reply by wendi881** [reply]
Thank you so much for your explanation! I will start with the Wiki text. I truly appreciate your support!

**Reply by wendi881** [reply]
Hi Digisembler, I do have a question. I don't know much about 3-D printers. If I am able to get funds for this project, I will need to buy a printer. Is there anything I need to know about size or type of 3-D printer to be able to print all of the parts of the ISS Mimic? Do you have any recommendations from your experience with 3-D printers?

**Reply by Digisembler** [reply]
Sure thing - and, whew that's a loaded question. ;) Everyone has their favorite 3d printers.  We have had great success with [Prusa Mini+ machines](https://www.prusa3d.com/product/original-prusa-mini-semi-assembled-3d-printer-7/), and all of ISS MIMIC will print on that small print bed (you'll need to use the 2 piece solar arrays on any print bed smaller than ~350x350 mm like the Ender 5 Plus has (~13.75" in the long axis), but they will work).  [All3dp.com](https://all3dp.com/1/best-3d-printer-reviews-top-3d-printers-home-3-d-printer-3d/) usually has a ranked 3d printer listing for favorites/comparison, I'm sure there are other sites that do too, and there are many youtubers that review 3d printers (like [3dPrintingNerd](https://www.youtube.com/@3DPrintingNerd)).  There are 3d printer companies that have school packages/discounts too based on your budget and school's/district's capabilities.  We do work with [re:3d](https://re3d.org/) who can print the whole kit.  Certainly ok with continued conversations as you work to decide/proceed.  We can move this over to our [Discord channel](https://discord.com/channels/764217406041882684/764217406041882687) too if you'd rather/are familiar with that.

**Reply by wendi881** [reply]
Hi Digisembler,
Thank you for your help with the 3D printers. I applied for a grant so I might get to build one!
I have more questions for you. I joined Discord, but I don't know how to find you/ your group there. Am I looking for a particular server? I have never used Discord before. Thank you for your help!

**Reply by ISS-Mimic** [reply]
Hi, this link will take you to our discord: https://discord.gg/zPKyE6hBSe

**Comment by wendi881** [comment]
Thank you! I can't wait to see the video you guys just put on there!


Thanks,

Mrs. Wendi Martinez

8th Grade Honors and On Level Science Teacher
Science Fair Cosponsor
FLL Robotics Team Coach

Rice Middle School
8500 Gifford Drive
Plano, TX 75025
p: 469-752-6000 ext. 26061
w: www.pisd.edu/rice<http://www.pisd.edu/rice>

[Facebook icon]<https://www.facebook.com/PlanoISD>          [Twitter icon] <https://twitter.com/i/flow/login?redirect_after_login=%2FPlano_Schools>           [Instagram icon] <https://www.instagram.com/planoschools/>

[Rice Middle School logo]




________________________________

**Comment by ISS-Mimic** [comment]
1/10/2024: Tag up summary

1. Discussed work that needs to be done for the museum build #503 
2. Reviewed actions - @Digisembler agreed to add a wire pass-through hole in the lofi P6 LS models and upload current S1 and P1 models to github, and investigate if anyone wants to help out with the code
3. @ayasname agreed to work on the MRM2 model hifification 
4. We need to reorganize the wiki

**Comment by juggler2000** [comment]
1/24/2024 tagup
@Digisembler & @juggler2000 

Talked a few issues, added comments.

**Comment by Digisembler** [comment]
1/31/2024 - @Digisembler, @SpaceTowel, @juggler2000, @streadgold, @dkpearland. Discussed SEEC logistics (2/7-9), SIS STEM night March 7th, SCH Moon2Mars support (mid march).  Issue discussion - several issues.

**Comment by SpaceTowel** [comment]
2/28/2024 _  @SpaceTowel, @juggler2000, @streadgold, @dkpearland, @ayasname , @GodspeedDiscvry .  Discussed SIS STEM night March 7th; SCH Moon2Mars; and a path to get to update Crew and VV screens without each user required to manual git pull.

**Comment by GodspeedDiscvry** [comment]
Also USOS and ROS screens.

**Comment by SpaceTowel** [comment]
- FIRST Robotic event for week of 4/17 to 4/19 planning. 
- mini potential mod to accommodate LED , add Prichard + swap out Z1
- when to do motor revamp , will they be needed in time for summer institute ?
